service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -263,6 +291,73 @@ void shouldAttemptGrantAccessToAllLocalAuthorityUsersWhenGrantAccessFailsForSome
         postSubmittedEvent(getCase(LA_NOT_IN_PRD_CODE));
 
         verifyGrantCaseRoleAttempts(LA_NOT_IN_PRD_USER_IDS, 2);
+        verifyNoInteractions(caseDataAccessApi);
+    }
+
+    @Test
```

Comment:
if you want  you can wrap all test with toggle false in @Nested class, and same for toggle on

Reply:
as discussed , we tried it but nested has issues, hence reverting that change and initialising that instance in individual tests.